packages/botonic-react/src/components/button.jsx

```diff
@@ -149,14 +166,26 @@ export const Button = props => {
           color: buttonTextColor,
           backgroundColor: buttonBgColor,
           ...(props.disabled && autoDisable && disabledStyle),
+          position: 'relative',
         }}
         bottom={props.bottomRadius}
       >
         {props.children}
+        {props.url && urlIcon && (
+          <img
+            className='botonic-url-icon'
+            style={{ width: '20px', position: 'absolute', right: '12px' }}
```

Comment:
If you set the prop position absolute, the image is above the button text

Reply:
I wouldn't use inline styles, these styles can't be easily overwritten and are the ones we end up putting an !important. 
Better to use css or styles in a styled component